### PR TITLE
feat(events-v2): Add function to transform events query for API

### PR DESCRIPTION
**Pull request by lynnagara**
Converts the columns specified by a view into a list of fields to be
requested.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationEventsV2/table.jsx [hunk 1]
```diff
@@ -47,8 +48,8 @@ class Table extends React.Component {
       return <LoadingIndicator />;
     }
 
-    return events.map(event => (
-      <PanelItem key={event.id} className={getGridStyle(fields.length)}>
+    return events.map((event, idx) => (
+      <PanelItem key={idx} className={getGridStyle(fields.length)}>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/table.jsx]: are events not unique in the table? when would we have duplicated events?

File: src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx [hunk 2]
```diff
@@ -64,3 +73,37 @@ export const ALL_VIEWS = [
 export function getCurrentView(requestedView) {
   return ALL_VIEWS.find(view => view.id === requestedView) || ALL_VIEWS[0];
 }
+
+/**
+ * Fetch organization events given view object
+ *
+ * @param {Object} api
+ * @param {String} orgSlug
+ * @param {Object} data
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx]:
```suggestion
 * @param {Object} view
```

File: src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx [hunk 3]
```diff
@@ -64,3 +73,37 @@ export const ALL_VIEWS = [
...
+  });
+}
+
+/**
+ * Takes a view and converts it into the format required for the events API
+ *
+ * @param {Object} view
+ * @returns {Object}
+ */
+export function getQuery(view) {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx]: Might be helpful here to describe the shape of `view` here

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationEventsV2/table.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationEventsV2/table.jsx]: We won't have duplicated events but a row might now represent a single event In the errors and csp tab each row is a grouping of events and hence has no event id associated

**Pull request merged**